src/search/landmarks/landmark_factory.cc

```diff
@@ -70,14 +70,14 @@ shared_ptr<LandmarkGraph> LandmarkFactory::compute_lm_graph(
     // the following replaces the old "build_lm_graph"
     generate(task_proxy, exploration);
     utils::g_log << "Landmarks generation time: " << lm_generation_timer << endl;
-    if (lm_graph->number_of_landmarks() == 0)
+    if (lm_graph->get_num_landmarks() == 0)
         utils::g_log << "Warning! No landmarks found. Task unsolvable?" << endl;
     else {
-        utils::g_log << "Discovered " << lm_graph->number_of_landmarks()
-                     << " landmarks, of which " << lm_graph->number_of_disj_landmarks()
+        utils::g_log << "Discovered " << lm_graph->get_num_landmarks()
+                     << " landmarks, of which " << lm_graph->get_num_disjunctive_landmarks()
                      << " are disjunctive and "
-                     << lm_graph->number_of_conj_landmarks() << " are conjunctive." << endl;
-        utils::g_log << lm_graph->number_of_edges() << " edges" << endl;
+                     << lm_graph->get_num_conjunctive_landmarks() << " are conjunctive." << endl;
+        utils::g_log << lm_graph->get_num_edges() << " edges" << endl;
     }
     //lm_graph->dump();
```

Comment:
Since the `dump` method is called differently now, we should probably either remove or change this call to it, even though it is commented out.

Reply:
I removed the call. Should be easy enough to add back in if we ever need it.